src/libs/ReportUtils.ts

```diff
@@ -3525,6 +3603,11 @@ function shouldReportBeInOptionList(report: OnyxEntry<Report>, currentReportId:
         return true;
     }
 
+    // Always show IOU reports with violations
+    if (isExpenseRequest(report) && transactionThreadHasViolations(report, betas.includes(CONST.BETAS.VIOLATIONS), transactionViolations, allReportActions)) {
```

Comment:
This is the alternative I alluded to above, about checking `canUseViolations` before calling `transactionThreadHasViolation` and not passing it as a param. 

Can we use `canUseViolations` instead of `betas.includes(CONST.BETAS.VIOLATIONS)`?
```suggestion
    if (isExpenseRequest(report) && canUseViolations && transactionThreadHasViolations(report, transactionViolations, allReportActions)) {
```

Reply:
Yeah, that makes sense to check for access to betas beforehand, good catch. In this specific instance though, we already had access to the betas, where if we were to use `canUseViolations` we'll have to call the `usePermissions` hook in a parent component since hooks can't be called outside of a React component.